<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>物业系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>48</number>
        </property>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QPushButton" name="btnChangeStaff">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;管理员工&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtChangeStaff">
            <property name="text">
             <string>管理员工</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QPushButton" name="btnPassVacation">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;批假&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtPassVacation">
            <property name="text">
             <string>批假</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QPushButton" name="btnRepair">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;报修&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtRepair">
            <property name="text">
             <string>报修</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QPushButton" name="btnChangeOwner">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;业主管理&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtChangeOwner">
            <property name="text">
             <string>业主管理</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="6">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>48</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="btnAddStaff">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;添加员工&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtAddStaff">
            <property name="text">
             <string>添加员工</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QPushButton" name="btnAttend">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;出勤登记&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtAttend">
            <property name="text">
             <string>出勤登记</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QPushButton" name="btnAddCar">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;车位添加&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtAddCar">
            <property name="text">
             <string>车位添加</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QPushButton" name="btnAppCar">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;车位申请&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtAppCar">
            <property name="text">
             <string>车位申请</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="5">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>48</number>
        </property>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <item>
           <widget class="QPushButton" name="btnPassCar">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;车位出租&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtPassCar">
            <property name="text">
             <string>车位出租</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <item>
           <widget class="QPushButton" name="btnAddOwner">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;业主登记&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtAddOwner">
            <property name="text">
             <string>业主登记</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_17">
          <item>
           <widget class="QPushButton" name="btnReSchedule">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;维修进度&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtReSchedule">
            <property name="text">
             <string>维修进度</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPerson">
    <property name="title">
     <string>人事管理</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>维护</string>
     </property>
     <addaction name="action_18"/>
     <addaction name="action_19"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menuOwner">
    <property name="title">
     <string>业主管理</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menuCar">
    <property name="title">
     <string>车位管理</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>车位信息</string>
     </property>
     <addaction name="action_23"/>
     <addaction name="action_24"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
   </widget>
   <widget class="QMenu" name="menuRepair">
    <property name="title">
     <string>维修管理</string>
    </property>
    <addaction name="action_12"/>
    <addaction name="action_15"/>
   </widget>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>系统设置</string>
    </property>
    <addaction name="action_9"/>
    <addaction name="action_13"/>
    <addaction name="action_17"/>
    <addaction name="action_16"/>
    <addaction name="action_21"/>
   </widget>
   <addaction name="menuPerson"/>
   <addaction name="menuOwner"/>
   <addaction name="menuCar"/>
   <addaction name="menuRepair"/>
   <addaction name="menuSystem"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_24">
   <property name="text">
    <string>查询</string>
   </property>
  </action>
  <action name="action_23">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/IconPic/pic/addCar.png</normaloff>:/IconPic/pic/addCar.png</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="action_22">
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>注销</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>修改</string>
   </property>
  </action>
  <action name="action_19">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/IconPic/pic/changeStaff.png</normaloff>:/IconPic/pic/changeStaff.png</iconset>
   </property>
   <property name="text">
    <string>查找/修改</string>
   </property>
  </action>
  <action name="action_18">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/IconPic/pic/addStaff.png</normaloff>:/IconPic/pic/addStaff.png</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>系统初始化</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="action_15">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/IconPic/pic/reSchedule.png</normaloff>:/IconPic/pic/reSchedule.png</iconset>
   </property>
   <property name="text">
    <string>物业维修进度</string>
   </property>
  </action>
  <action name="action_12">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/IconPic/pic/repair.png</normaloff>:/IconPic/pic/repair.png</iconset>
   </property>
   <property name="text">
    <string>报修</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/IconPic/pic/passCar.png</normaloff>:/IconPic/pic/passCar.png</iconset>
   </property>
   <property name="text">
    <string>车位出租</string>
   </property>
  </action>
  <action name="action_10">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/IconPic/pic/appCar.png</normaloff>:/IconPic/pic/appCar.png</iconset>
   </property>
   <property name="text">
    <string>车位申请</string>
   </property>
  </action>
  <action name="action_8">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/IconPic/pic/changeOwner.png</normaloff>:/IconPic/pic/changeOwner.png</iconset>
   </property>
   <property name="text">
    <string>业主查询</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/IconPic/pic/addOwner.png</normaloff>:/IconPic/pic/addOwner.png</iconset>
   </property>
   <property name="text">
    <string>业主登记</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/IconPic/pic/staffAttend.png</normaloff>:/IconPic/pic/staffAttend.png</iconset>
   </property>
   <property name="text">
    <string>出勤登记</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/IconPic/pic/passVacation.png</normaloff>:/IconPic/pic/passVacation.png</iconset>
   </property>
   <property name="text">
    <string>批假</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>销假</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>请假</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>维护</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>出勤状况</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myimages.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_24</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actFindCar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_23</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actAddCar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_21</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_19</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actFindStaff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_18</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actAddPerson()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_15</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actReSchedule()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_12</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actNeedRepair()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_11</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actRentCar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_10</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actAppCar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_8</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actFindOwner()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_6</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actAddOwner()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_5</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actAttend()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_4</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actPassVaca()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_3</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actCancelVaca()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_2</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actVacation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_17</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actInit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_16</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_7</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actAttendState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_9</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actSetup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_13</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actChPassword()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cancel()</slot>
  <slot>actAddPerson()</slot>
  <slot>actFindStaff()</slot>
  <slot>actVacation()</slot>
  <slot>actPassVaca()</slot>
  <slot>actCancelVaca()</slot>
  <slot>actAddOwner()</slot>
  <slot>actFindOwner()</slot>
  <slot>actAddCar()</slot>
  <slot>actFindCar()</slot>
  <slot>actAppCar()</slot>
  <slot>actRentCar()</slot>
  <slot>actNeedRepair()</slot>
  <slot>actReSchedule()</slot>
  <slot>actAttend()</slot>
  <slot>actInit()</slot>
  <slot>actHelp()</slot>
  <slot>actAttendState()</slot>
  <slot>actSetup()</slot>
  <slot>actChPassword()</slot>
 </slots>
</ui>
